lib/Simulator/signals-runtime-wrappers.cpp

```diff
@@ -0,0 +1,76 @@
+#include "circt/Simulator/signals-runtime-wrappers.h"
+#include "circt/Simulator/State.h"
+
+#include "llvm/ADT/APInt.h"
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/Support/raw_ostream.h"
+
+#include <cassert>
```

Comment:
Please drop redundant #includes where possible, certainly cassert can be dropped, I suspect ArrayRef as well, possibly others